### PR TITLE
[RFC] Temporary disable code which references mch_resolve_shortcut

### PR DESCRIPTION
**Pull request by equalsraf**
For now we don't a have working implementation of mch_resolve_shortcut. We can revisit this at a later time, for now just the code.

Picked from #810

### CODE REVIEW

File: src/nvim/buffer.c [hunk 1]
```diff
@@ -3994,7 +3994,9 @@ void fname_expand(buf_T *buf, char_u **ffname, char_u **sfname)
     *sfname = *ffname;
   *ffname = (char_u *)fix_fname((char *)*ffname);     /* expand to full path */
 
-#ifdef WIN32
+// TODO(Hettomei,equalsraf): Temporary disable this block
```
Comment (brcolow) [inline src/nvim/buffer.c]: Temporary => Temporarily

**Comment by equalsraf** [comment]
Fixed typos

**Comment by justinmk** [comment]
Any reason not to just leave this in #810 until we have an actual implementation?

**Comment by equalsraf** [comment]
No particular reason. Cleaning up, trying to get all commits that block the build out of #810.

**Comment by justinmk** [comment]
https://github.com/neovim/neovim/pull/4837 is a WIP to restore mch_resolve_shortcut.

**Pull request closed without merging**